Library/Formula/sdl_ttf.rb

```diff
@@ -18,6 +18,10 @@ class SdlTtf < Formula
   depends_on "sdl"
   depends_on "freetype"
 
+  # Fix broken TTF_RenderGlyph_Shaded()
+  # https://bugzilla.libsdl.org/show_bug.cgi?id=1433
```

Comment:
How do you feel about this @mikemcquaid? It's likely to never ever be fixed upstream sadly. I guess we either carry the patch, keep the slightly broken formula, or remove the formula entirely.